script.js

```diff
@@ -0,0 +1,165 @@
+const toDoForm = document.querySelector(".todo-form");
+const toDoInput = toDoForm.querySelector("input");
+const toDoInputBtn = document.querySelector(".todo-input").querySelector("div");
```

Comment:
```suggestion
const toDoInputBtn = document.querySelector(".todo-input div");
```
css 선택자 문법을 `querySelector`에 그대로 적용할 수 있습니다.

Reply:
지적해 주신 부분 덕분에 한번 더 `querySelector` 의 사용 목적에 대해 짚고 넘어가게 되었습니다ㅎㅎ
감사드립니다.